resources/eventing/charts/nats/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: nats
+description: Helm chart for nats
+type: application
+version: 0.1.0
+appVersion: 0.1.0
```

Comment:
(nit): f you would like to use appVersion, it should have the same version as nats and we should commit to keep it up2date :)

> appVersion: The version of the app that this contains (optional). This needn't be SemVer.
https://helm.sh/docs/topics/charts/

Reply:
removed